### DISCUSSION TITLE
Is there a way to turn off view bobbing/camera shake in first person camera?

### DISCUSSION THREAD

**Discussion by sashakoshka** [raysan5/raylib#1990] [Q&A]
View bobbing seems to be on by default in the first person camera. Is there a way to turn this off?

**Comment by Safariminer** [comment]
Commenting those lines in camera.h disables bobbing. I'm still actively looking how to disable the slight camera tilt when you move.
![image](https://user-images.githubusercontent.com/74744603/147774054-fbc38586-07a3-466a-bb9a-3b44b396617c.png)
